static/missing_avatar.svg

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Using https://jakearchibald.github.io/svgomg/ we can get the size down I think (eg. inkscape annotations)

See if the following is equivalent:
```
<svg xmlns="http://www.w3.org/2000/svg" width="120mm" height="453.543" viewBox="0 0 120 120">
  <g stroke="#000" stroke-linecap="round">
    <path opacity=".99" fill="#e6e6e6" fill-opacity=".246" stroke-width=".065" d="M.253.032h119.494v119.935H.253z"/>
    <path d="M119.967 60a59.967 59.967 0 01-59.715 59.967A59.967 59.967 0 01.035 60.505 59.967 59.967 0 0159.243.038 59.967 59.967 0 01119.96 58.99" opacity=".99" fill="#999" fill-opacity=".771" stroke-width=".065"/>
    <path d="M59.653 18.575a27.466 27.466 0 00-27.117 27.694 27.466 27.466 0 0011.047 21.76 37.469 37.469 0 00-21.05 33.944 37.469 37.469 0 00.438 5.117 59.967 59.967 0 002.95 2.223 34.971 34.971 0 01-.89-7.36 34.971 34.971 0 0120.986-32.307 27.466 27.466 0 0014.098 3.857 27.466 27.466 0 0013.872-3.861 34.971 34.971 0 0120.979 31.428l.005.588a34.971 34.971 0 01-.858 7.578 59.967 59.967 0 002.922-2.09 37.469 37.469 0 00.434-5.488l-.005-.63a37.469 37.469 0 00-21.011-33.032 27.466 27.466 0 01-.069.047 27.466 27.466 0 0011.082-22.005l-.004-.462a27.466 27.466 0 00-27.809-27.001zm.032 2.496a24.969 24.969 0 0125.28 24.547l.004.42a24.969 24.969 0 01-24.864 24.969 24.969 24.969 0 01-25.073-24.759 24.969 24.969 0 0124.653-25.177z" opacity=".99" fill="#e6e6e6" stroke-width=".0687908"/>
  </g>
</svg>
```

Reply:
updated! had to leave viewbox sizing in, but all standard options left on